tests/tests_integration/test_api/test_transformations/test_transformations.py

```diff
@@ -407,6 +407,7 @@ def test_update_instance_nodes(self, cognite_client, new_transformation):
             ViewInfo("myViewExternalId", "myViewVersion2", "test-space"), "test-space"
         )
 
+    @pytest.mark.skip  # TODO: view and edgeType are mutually exclusive (code: 400)
```

Comment:
```suggestion
    @pytest.mark.xfail  # TODO: view and edgeType are mutually exclusive (code: 400)
```

Reply:
I guess this was already fixed in a separate PR. I'll verify.

Reply:
Also this now: https://github.com/cognitedata/cognite-sdk-python/pull/1174